lldb/source/Plugins/LanguageRuntime/Swift/SwiftTask.cpp

```diff
@@ -0,0 +1,53 @@
+#include "SwiftTask.h"
+#include "SwiftLanguageRuntime.h"
+#include "lldb/Target/Process.h"
+
+using namespace llvm;
+using namespace lldb;
+
+lldb_private::ThreadTask::ThreadTask(tid_t tid, addr_t async_ctx,
+                                     ExecutionContext &exe_ctx)
+    : Thread(exe_ctx.GetProcessRef(), tid, true),
+      m_concrete_reg_ctx_sp(exe_ctx.GetFrameSP()->GetRegisterContext()) {
+  m_async_ctx = async_ctx;
```

Comment:
we can initialize `m_sync_ctx` in the member initializer list?

Reply:
I wanted like with like: to have the register value initialization logic done together.